### PR TITLE
Proposal for IDE maintenance

### PR DESCRIPTION
**Pull request by lawmeskiviahs**
This is an application for the open RFP: https://github.com/w3f/Grants-Program/blob/master/docs/RFPs/Open/IDE_for_ink_Smart_Contracts.md

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. What do you mean by 2 FTE 4 PTE? 11,200 every two weeks sounds relatively expensive to me. But I will also share the application with the rest of the team.

**Comment by lawmeskiviahs** [comment]
Thankyou for the reviewing the application. 2 FTE and 4 PTE means 2 full time and 4 part time. As you suggested, we have updated the costing. Kindly review the same.

**Comment by lawmeskiviahs** [comment]
Thankyou for reviewing the application. 2 FTE and 4 PTE means 2 full time
and 4 part time. As you suggested, we have updated the costing. Kindly
review the same.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the quick reply. This sounds like a lot for this project. We also discussed it internally, and it might make sense to apply for a regular grant to fix the following issues initially mostly:

- https://github.com/paritytech/ink-playground/issues/427
- https://github.com/paritytech/ink-playground/issues/197
- https://github.com/paritytech/ink-playground/issues/460
- https://github.com/paritytech/ink-playground/issues/428

Basically, making the playground compatible with different versions of ink! and automatic updates. Once this is done, we will sign a smaller maintenance agreement. We will also update the RFP accordingly.

**Comment by lawmeskiviahs** [comment]
Thankyou for the update. We have updated the changes, kindly review again.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the update and for reducing the price. But instead of submitting a maintenance grant, could you instead change it to a regular one time grant, see https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md

**Comment by lawmeskiviahs** [comment]
Will do.
